client/src/modules/stock/lots/registry.js

```diff
@@ -245,8 +221,15 @@ function StockLotsController(
 
     Stock.lots.read(null, filters)
       .then((lots) => {
+
+        // FIXME(@jniles): we should do this ordering on the server via an ORDER BY
+        lots
+          .sort(orderByDepot);
```

Comment:
Could you add this change also in the stock/inventory and stock/movement registries.